calculator.js

```diff
@@ -1,19 +1,67 @@
+// const prompt = require('prompt')
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+
+// const exampleAdditionInput = {
+//   num1: 3,
+//   num2: 5,
+//   operation: 'add',
+// }
+
+const add = function(num1, num2) {
+  console.log(`${num1} + ${num2} = ${Number(num1) + Number(num2)}`)
```

Comment:
It would have been nice to parse the input into `Number`s one time before passing them to the functions.